distributed/tests/test_worker.py

```diff
@@ -183,18 +183,13 @@ def g():
 
 @gen_cluster(client=True)
 def test_upload_egg(c, s, a, b):
-    eggname = 'mytestegg-1.0.0-py3.4.egg'
+    eggname = 'testegg-1.0.0-py3.4.egg'
     local_file = __file__.replace('test_worker.py', eggname)
     assert not os.path.exists(os.path.join(a.local_dir, eggname))
     assert not os.path.exists(os.path.join(b.local_dir, eggname))
     assert a.local_dir != b.local_dir
 
-    aa = rpc(a.address)
-    bb = rpc(b.address)
-    with open(local_file, 'rb') as f:
-        payload = f.read()
-    yield [aa.upload_file(filename=eggname, data=payload),
-           bb.upload_file(filename=eggname, data=payload)]
+    yield c.upload_file(filename=eggname)
```

Comment:
From the test failures it looks like this should maybe be `local_file` instead?  It seems like it's having trouble finding the `eggname` file.  I'm curious, are you able to run tests locally and, if so, do these tests work locally for you?